### PR TITLE
Home Location UX

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#12962

*Description of changes:*
* UX Improvements to Home Location Land Type Component, see ticket for details.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]
Looks good, nice and easy. Package conflict which I am sure you saw.

**Comment by cameron-eyds** [comment]
> Looks good, nice and easy.

haha thank you!

File: ppr-ui/src/components/common/PidNumber.vue [hunk 1]
```diff
@@ -89,6 +95,17 @@ export default defineComponent({
       })
     })
 
+    /** Handle pasted event into Pid Input Fields **/
+    const parsePaste = (event: any): void => {
+      // Capture pasted text from event and clean spaces/special chars
+      const pasteInput = event.clipboardData.getData('text').replace(/[^A-Z\d]+/ig, '')
```
Comment (dimak1) [inline ppr-ui/src/components/common/PidNumber.vue:101]: Nice implementation! 
Do we need to check that length is 9 before pasting?

**Review by dimak1 — APPROVED** [review]
LGTM! Just one comment about the length of PID before pasting.

File: ppr-ui/src/components/common/PidNumber.vue (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/components/common/PidNumber.vue:101]: Wasn't really specified, will discuss with Tracey during UX. Allowing a copy paste of over 9 chars and taking the first 9 seems like a better UX than just preventing the paste.
But we will see!

**Pull request merged**